letsencrypt/views.py

```diff
@@ -27,11 +24,5 @@ def detail(request, acme_data):
         AcmeChallenge,
         challenge=acme_data,
     )
-
-    return render(
-        request,
-        'letsencrypt/detail.html',
-        {
-            'response': acme_challenge.response
-        },
-    )
+    
```

Comment:
I believe you have some bad whitespace here you shouldn't have since your build failed: https://travis-ci.org/urda/django-letsencrypt/builds/205937956

You should be able to do the following to test the project locally:

1. Create a `virtualenv` for any of the supported python versions
2. `pip install -r requirements.txt`
3. `make test` should run completely with no errors.

I'll eventually update the project documents to list this out for future contributions.